src/components/review/StorageReview.jsx

```diff
@@ -214,65 +214,119 @@ const isDeviceDeleted = ({ actions, device }) => (
     actions.find(action => action["device-name"].v === device && action["action-type"].v === "destroy")
 );
 
-export const StorageReviewNote = () => {
-    const plannedActions = usePlannedActions();
+/**
+ * @returns {boolean}   True is the device will be resized according to the actions
+ */
+const isDeviceResized = ({ actions, device }) => (
+    actions.find(action => action["device-name"].v === device && action["action-type"].v === "resize")
+);
+
+const DeletedSystems = () => {
     const originalExistingSystems = useOriginalExistingSystems();
-    const originalDevices = useOriginalDevices();
+    const plannedActions = usePlannedActions();
 
     const deletedSystems = originalExistingSystems.filter(
         system => system.devices.v.every(device => isDeviceDeleted({ actions: plannedActions, device }))
     );
+
+    return deletedSystems.map(system => (
+        <ListItem key={system["os-name"].v}>
+            {cockpit.format(_("$0 will be deleted"), system["os-name"].v)}
+        </ListItem>
+    ));
+};
+
+const AffectedSystems = ({ type }) => {
+    const originalExistingSystems = useOriginalExistingSystems();
+    const originalDevices = useOriginalDevices();
+    const plannedActions = usePlannedActions();
+
+    const check = device => (
+        type === "delete"
+            ? isDeviceDeleted({ actions: plannedActions, device })
+            : isDeviceResized({ actions: plannedActions, device })
+    );
+
     const affectedSystems = originalExistingSystems.filter(
-        system => (
-            system.devices.v.some(device => isDeviceDeleted({ actions: plannedActions, device })) &&
-            !system.devices.v.every(device => isDeviceDeleted({ actions: plannedActions, device }))
-        )
+        system => {
+            const systemDevices = system.devices.v;
+
+            return (
+                systemDevices.some(check) &&
+                !systemDevices.every(check)
```

Comment:
If all the devices of a system are resized (not saying it makes sense), would the system be reported as affected?

Reply:
Oh it will not. I had this check for 'deleted systems', I report in the review screen systems which were comletely wiped out differently than ones with partitially deleted partitions. 

For resize this is not interesting. I guess I need to adjust the (!systemDevices.every) to do that only for deleted.

Reply:
Fixed